detectron2/modeling/meta_arch/retinanet.py

```
@@ -353,11 +353,16 @@ def norm(c):
                 )
 
         else:
-            norm_name = str(type(get_norm(norm, 1)))
-            if "BN" in norm_name:
-                logger.warning(
-                    f"Shared BatchNorm (type={norm_name}) may not work well in RetinaNetHead."
-                )
+            try:
+                norm_name = str(type(get_norm(norm, 1)))
+                if "BN" in norm_name:
+                    logger.warning(
+                        f"Shared BatchNorm (type={norm_name}) may not work well in RetinaNetHead."
+                    )
+            except ValueError:
+                # https://github.com/pytorch/pytorch/pull/74293
+                if isinstance(norm, str):
+                    logger.warning(f"No BatchNorm was found for '{norm}' for RetinaNetHead.")
```

Please undo this change. There is no reason to ignore the exception.